opcua/common/xmlexporter.py

```diff
@@ -276,11 +282,16 @@ def value_to_etree(el, dtype_name, dtype, node):
 
 def _value_to_etree(el, dtype_name, dtype, val):
     if isinstance(val, (list, tuple)):
-        list_el = Et.SubElement(el, "uax:ListOf" + dtype_name)
-        for nval in val:
-            _value_to_etree(list_el, dtype_name, dtype, nval)
```

Comment:
Do you mean make the "ListofX" element in `value_to_etree`? You typed `_value_etree` twice.